cupyx/scipy/sparse/csc.py

```diff
@@ -305,7 +305,7 @@ def getrow(self, i):
         Returns:
             cupyx.scipy.sparse.csc_matrix: Sparse matrix with single row
         """
-        return self._minor_slice(slice(i, i + 1), copy=True)
+        return self.tocsr()._major_slice(slice(i, i + 1), copy=True)
```

Comment:
Is it faster than `self._minor_slice(slice(i, i + 1), copy=True).tocsr()`?

BTW, I was surprised at output types:

| | `getrow` | `getcol` |
| - | - | - |
| `csr_matrix` | `csr_matrix` | `csr_matrix` |
| `csc_matrix` | `csr_matrix` | `csc_matrix` |

Reply:
I think `_major_slice` is faster than `_minor_slice`, but not compared the performance yet, I'll show the performance numbers later.

Reply:
`_major_slice` is faster than `_minor_slice` actually. However, I compared the performance and turned out that converting to csr format after slicing is faster. I fixed in b73ca10.